build-simulation.py

```diff
@@ -75,11 +126,10 @@ def generate_list(values, t, num_instances, num_markets):
 raw_sp_input = pd.read_csv('data/input_sp.csv')
 instances = raw_input['instance'].value_counts()
 
-if checkInputSP(raw_sp_input, instances) == False:
-    raise ValueError("Error in the savings plan input.")
-
 raw_demand = pd.read_csv('data/TOTAL_demand.csv')
 
+validate_input(raw_input, raw_sp_input, raw_demand)
```

Comment:
uhm ...
i think the code would be better if we have three functions here:
1. validate_raw_input
2. validate_sp_input (what is sp btw? it is not clear)
3. validate_demand

Reply:
of course, each one of these methods will call their own inner different functions

Reply:
Ok.

Reply:
sp means savings plan, i will add this in the doc.